concert/app/src/functions/on_event_stage_changed/index.py

```diff
@@ -88,3 +104,107 @@ def get_song_list_update_transaction_item(message_data):
     }
 
     return {"Update": song_list_update_args}
+
+
+def get_song_list_get_transaction_item():
+    serializer = boto3.dynamodb.types.TypeSerializer()
+    song_list_get_args = {
+        "Key": {ATTR_RECORD_ID: serializer.serialize(RECORD_ID_SONG_LIST)},
+        "TableName": DYNAMODB_TABLE_NAME,
+    }
+
+    return {"Get": song_list_get_args}
+
+
+def get_aggregate_data_get_transaction_item():
+    serializer = boto3.dynamodb.types.TypeSerializer()
+    aggregate_data_get_args = {
+        "Key": {ATTR_RECORD_ID: serializer.serialize(RECORD_ID_AGGREGATE)},
+        "TableName": DYNAMODB_TABLE_NAME,
+    }
+
+    return {"Get": aggregate_data_get_args}
+
+
+def get_song_list_and_aggregate_data():
+    song_list_get_transaction_item = get_song_list_get_transaction_item()
+    aggregate_data_get_transaction_item = get_aggregate_data_get_transaction_item()
+
+    transaction_items = [
```

Comment:
Style: `song_list_get_transaction_item`, `aggregate_data_get_transaction_item`, and `transaction_items` are each only used once. You could make this easier to read with
```py
responses = dynamodb.transact_get_items(TransactItems=[
    get_transaction_item(RECORD_ID_SONG_LIST),
    get_transaction_item(RECORD_ID_AGGREGATE),
])["Responses"]
```

Reply:
Done.